src/main/java/com/example/codeE/CodeEApplication.java

```diff
@@ -6,7 +6,7 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-@EnableSwagger2
+//@EnableSwagger2
```

Comment:
LOL

Reply:
swagger dependency is having some errors now, WW-116 will resolve that.